Branch-SDK/src/io/branch/referral/SystemObserver.java

```diff
@@ -324,7 +324,8 @@ public boolean getBluetoothPresent() {
         try {
             BluetoothAdapter bluetoothAdapter = BluetoothAdapter.getDefaultAdapter();
             if (bluetoothAdapter != null) {
-                return bluetoothAdapter.isEnabled();
+				//noinspection ResourceType
```

Comment:
Hehe. I figured. Starting Android M this code actually needs to be modified again, so someone should probably put that on a sticky!